### DISCUSSION TITLE
Discord is not working (for me)?

### DISCUSSION THREAD

**Discussion by jrohrauer** [snipe/snipe-it#15348] [General]
Greetings,

I am not sure if this is the right place. I wanted to ask a question regardig upgrade/migration to a new server, but when I try to open Discord I see the invitation, I click on it and then I get a generic error like "Invitation could not be accepted" (something like this, message is in german).

Is this a generel error or something on my end?

Regards
Johannes

**Comment by snipe** [comment]
Huh - we have tons of people in the discord, so I'm not sure what might be bonked. Can you try this one? https://discord.gg/WyGKDBhfYP

**Reply by jrohrauer** [reply]
Thanks, that link worked now without any issues :)
I am still not sure if this was my fault. FYI, I went to your website, went for free support and klicked the Discord link on the right side.

**Reply by snipe** [reply]
I'll check the link - maybe it expired or something. Thanks for the heads up, and welcome to the Discord. Lovely people in there. We wrote about it here: https://grokstar.dev/culture/2024/06/the-unlikely-rise-of-discord-as-a-support-channel/
